developing/code-reviews/great-code-reviews.md

```diff
@@ -0,0 +1,45 @@
+# Great Code Reviews Bring Happiness to my Heart
+
+So you’ve put together the perfect changeset. You use your favorite tool to share your changes and ask politely for a code review.
+
+How do you make that code review great?
+
+## Making code review process work for your team
+
+We've all been on teams large and small that have tried to integrate code review into their process. Some of those teams succeeded, and others failed. There were several themes that arose that were indicators for success or failure. Here are the practices we've found that tend to make code reviews a success.
+
+1. **Do it asynchronously.** If you or the reviewer are at all busy, and you probably are, do it asynchronously. Let the reviewer know there’s an item awaiting their attention, then give them a day to get to it. If they don’t get to it in a day, bug them again.
+
+2. **Use a visual code review management tool.** Use a code review tool which keeps track of comments on changes, and allows in-line discussion. In addition to GitHub’s code review tool there is Google's [Gerrit](https://www.gerritcodereview.com/), [Phabricator](https://phacility.com/phabricator/) that started as a Facebook project, or [JetBrains Upsource](https://www.jetbrains.com/upsource) as a paid option.
+
+3. **Make code review a gatekeeper step in your source management process.** If it’s something that must pass before code can be merged, then it will happen. GitHub’s pull request feature does a great job of supporting an integrated code review step. A dashboard shows what pull requests are outstanding, and a merge button minimizes friction once the changes have passed review. If you try to add code review as an adjunct step to an existing process - e.g. sending an email asking for a code review, with no central dashboard where outstanding review requests can be seen - it can be very difficult to consistently employ code review, since its existence will be somewhat invisible. YMMV.
```

Comment:
What does "YMMV" mean?

Reply:
It means Your Mileage May Vary, basically it may not work for everyone.